pkg/controller/externalippool/ipallocator/allocator.go

```diff
@@ -0,0 +1,253 @@
+// Copyright 2021 Antrea Authors
```

Comment:
this is duplicate with pkg/controller/externalippool/ipallocator/allocator.go

Reply:
deleted.